example/js/lib/api/ICU4XFixedDecimalGroupingStrategy.js

```diff
@@ -1,15 +1,23 @@
 import wasm from "../wasm.mjs"
-import * as diplomatRuntime from "./diplomat-runtime.mjs"
+import * as diplomatRuntime from "./diplomat-runtime.js"
 
 export const ICU4XFixedDecimalGroupingStrategy_js_to_rust = {
   "Auto": 0,
   "Never": 1,
   "Always": 2,
   "Min2": 3,
 };
+
 export const ICU4XFixedDecimalGroupingStrategy_rust_to_js = {
   0: "Auto",
   1: "Never",
   2: "Always",
   3: "Min2",
 };
+
+export const ICU4XFixedDecimalGroupingStrategy = {
```

Comment:
Question: Why did this change? Stale diplomat output?

Reply:
yeah switching to cargo-make will improve the status quo here

Reply:
I added that in this PR. The reason for it was that the `.d.ts` file exported an enum type, and so the js side had to export some equivalent type. Maybe there's some other workaround? Not sure, but I think this is one of the reasons why unions of strings are preferred in TS over string enums,